libraries/AP_BattMonitor/AP_BattMonitor_SMBus_Maxell.h

```diff
@@ -15,6 +15,10 @@ class AP_BattMonitor_SMBus_Maxell : public AP_BattMonitor_SMBus
                              AP_BattMonitor::BattMonitor_State &mon_state,
                              AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev);
 
+    // generic SMBus batteries may not always report the cell voltages, as we
+    // the data is being read from manufacturer specific fields
```

Comment:
as we the data -> as the data

I also disagree with what is written here. This isn't a generic driver, it is the Maxell driver, so we know it reports the cell voltages.